scheduler/core/calculations/targetinfo.py

```diff
@@ -54,6 +54,7 @@ class TargetInfo:
     airmass: npt.NDArray[float]
     sky_brightness: npt.NDArray[SkyBackground]
     visibility_slot_idx: npt.NDArray[int]
+    visibility_slot_filter: npt.NDArray[float]
```

Comment:
Why the filter needs to be added again? as far as I can tell it was not being used

Reply:
I was getting an error in the collector if this wasn't defined.